appinventor/components/src/com/google/appinventor/components/runtime/TextBox.java

```diff
@@ -153,6 +154,16 @@ public void HideKeyboard() {
     imm.hideSoftInputFromWindow(view.getWindowToken(), 0);
   }
 
+    /**
+     * Request focus to current textbox.
+     */
+    @SimpleFunction(
+            description = "Sets textbox active."
+    )
+    public void RequestFocus() {
+        TextViewUtil.requestFocus(view);
```

Comment:
Why create a new function in TextViewUtil. Why not just do "view.requestFocus()"?